### ISSUE TITLE
Could not find a package configuration file provided by "hsa-runtime64"

### ISSUE THREAD

**Issue by izypd** [rocm-arch/python-pytorch-rocm#6]
Hello! I am using `yay -S rocm-dev rocm-utils rocm-libs rccl` then `yay -S python-pytorch-opt-rocm` to build pytorch in EndeavourOS. First command succeeded, next command failed! I am using arch4edu.
Error code as so:
```
***** Library versions from cmake find_package *****

CMake Error at /usr/share/cmake-3.18/Modules/CMakeFindDependencyMacro.cmake:47 (find_package):
  By not providing "Findhsa-runtime64.cmake" in CMAKE_MODULE_PATH this
  project has asked CMake to find a package configuration file provided by
  "hsa-runtime64", but CMake did not find one.

  Could not find a package configuration file provided by "hsa-runtime64"
  with any of the following names:

    hsa-runtime64Config.cmake
    hsa-runtime64-config.cmake

  Add the installation prefix of "hsa-runtime64" to CMAKE_PREFIX_PATH or set
  "hsa-runtime64_DIR" to a directory containing one of the above files.  If
  "hsa-runtime64" provides a separate development package or SDK, be sure it
  has been installed.
Call Stack (most recent call first):
  /opt/rocm/hip/lib/cmake/hip/hip-config.cmake:124 (find_dependency)
  cmake/public/LoadHIP.cmake:131 (find_package)
  cmake/public/LoadHIP.cmake:170 (find_package_and_print_version)
  cmake/Dependencies.cmake:1136 (include)
  CMakeLists.txt:469 (include)


-- Configuring incomplete, errors occurred!
See also "/home/ym/.cache/yay/python-pytorch-rocm/src/pytorch-1.6.0-rocm/build/CMakeFiles/CMakeOutput.log".
See also "/home/ym/.cache/yay/python-pytorch-rocm/src/pytorch-1.6.0-rocm/build/CMakeFiles/CMakeError.log".
Traceback (most recent call last):
  File "setup.py", line 732, in <module>
    build_deps()
  File "setup.py", line 311, in build_deps
    build_caffe2(version=version,
  File "/home/ym/.cache/yay/python-pytorch-rocm/src/pytorch-1.6.0-rocm/tools/build_pytorch_libs.py", line 54, in build_caffe2
    cmake.generate(version,
  File "/home/ym/.cache/yay/python-pytorch-rocm/src/pytorch-1.6.0-rocm/tools/setup_helpers/cmake.py", line 329, in generate
    self.run(args, env=my_env)
  File "/home/ym/.cache/yay/python-pytorch-rocm/src/pytorch-1.6.0-rocm/tools/setup_helpers/cmake.py", line 141, in run
    check_call(command, cwd=self.build_dir, env=env)
  File "/usr/lib/python3.8/subprocess.py", line 364, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['cmake', '-GNinja', '-DBUILD_PYTHON=True', '-DBUILD_TEST=True', '-DCMAKE_BUILD_TYPE=Release', '-DCMAKE_INSTALL_PREFIX=/home/ym/.cache/yay/python-pytorch-rocm/src/pytorch-1.6.0-rocm/torch', '-DCMAKE_PREFIX_PATH=/usr/lib/python3.8/site-packages', '-DNUMPY_INCLUDE_DIR=/usr/lib/python3.8/site-packages/numpy/core/include', '-DPYTHON_EXECUTABLE=/usr/bin/python', '-DPYTHON_INCLUDE_DIR=/usr/include/python3.8', '-DPYTHON_LIBRARY=/usr/lib/libpython3.8.so.1.0', '-DTORCH_BUILD_VERSION=1.6.0a0+b31f58d', '-DUSE_CUDA=OFF', '-DUSE_NUMPY=True', '-DUSE_ROCM=ON', '/home/ym/.cache/yay/python-pytorch-rocm/src/pytorch-1.6.0-rocm']' returned non-zero exit status 1.
==> 错误： 在 build() 中发生一个错误。
    正在放弃...
生成时出错: python-pytorch-rocm (python-pytorch-opt-rocm)
```

**Comment by acxz** [comment]
upstream issue: https://github.com/pytorch/pytorch/issues/44384

**Comment by izypd** [comment]
> upstream issue: [pytorch/pytorch#44384](https://github.com/pytorch/pytorch/issues/44384)
> 
> seems like pytorch 1.6 is not ready for rocm 3.7

Thanks! I am trying to set some environmental variables mentioned in pytorch/pytorch#44384 then run `yay -S python-pytorch-opt-rocm`

**Comment by izypd** [comment]
> upstream issue: [pytorch/pytorch#44384](https://github.com/pytorch/pytorch/issues/44384)
> 
> seems like pytorch 1.6 is not ready for rocm 3.7

Thanks! I am trying to set some environmental variables mentioned in pytorch/pytorch#44384 then run `yay -S python-pytorch-opt-rocm`

**Comment by acxz** [comment]
> Could you please add a AUR package using pytorch 1.7(master)?

pytorch master at the time of posting this also gives the same error

> Or just give me some advise to build pytorch 1.6 sucessfuly.

I could, I would incorporate the solution to the PKGBUILD, haha

> Are there other informations should I provide?

I can reproduce the issue so any information you can provide to me would be redundant.

**Comment by acxz** [comment]
patched with 323aec0

**Comment by izypd** [comment]
> patched with [323aec0](https://github.com/rocm-arch/python-pytorch-rocm/commit/323aec03bb5d659fa7d2585e08f228b81157600e)

Thank you for your patch, but I still can not build pytorch sucessfuly, please see rocm-arch/python-pytorch-rocm#8 for details.

**Comment by acxz** [comment]
Closing this issue as the patch in incorporated in later pytorch versions.
